ui-v2/src/components/ui/state-badge/index.tsx

```diff
@@ -38,12 +38,12 @@ const stateBadgeVariants = cva("gap-1", {
 
 export const StateBadge = ({
 	state,
-}: { state: components["schemas"]["State"] }) => {
+}: { state: Omit<components["schemas"]["State"], "id"> & { id?: string } }) => {
 	const Icon = ICONS[state.type];
 	return (
 		<Badge className={stateBadgeVariants({ state: state.type })}>
 			<Icon size={16} />
-			{state.name}
+			{state.name ?? state.type}
```

Comment:
If there isn't a state name, then we should probably capital-case the type for visual consistency.